modeling/03_Per_Sat_Model.Rmd

```diff
@@ -840,8 +853,8 @@ export_summary_8.start()
 
 export_summary_9 = (ee.batch.Export.table.toDrive(
   collection = allAreas_l9,
-  selectors = ['missDate', 'classified', 'cloud', 'openWater', 'lightNSSed','OSSed', 'dNSSed', 'other', 'area_ha'],
-  description = 'quick_gradientTreeBoost_landsat_9_stack_v2023-07-25',
+  selectors = ['missDate', 'classified', 'cloud', 'openWater', 'lightNSSed','OSSed', 'dNSSed', 'area_ha'],
+  description = 'quick_gradientTreeBoost_landsat_9_stack_v' + v_date,
```

Comment:
I'm a work in progress. If I had considered using {targets} this wouldn't be an issue... I especially didn't want to deal with the multi-language aspect of that here. I was able to do it for our Landsat Acquisition pipeline, as there was only one step with python (and it's the final step), otherwise you have to get REALLY creative. :)

Reply:
Oh yeah, that gets complicated. Loving the incremental move towards easier reproducibility. In a way, its slightly better than trying to implement it right from the start because you sort of know the pain points for the manual parts and can focus attention and energy on just the stickiest ones!